schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/convert/protocol-mapping.xsl

```diff
@@ -0,0 +1,55 @@
+<xsl:stylesheet version="2.0"
+                xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                xmlns:dct="http://purl.org/dc/terms/"
+                xmlns:dcat="http://www.w3.org/ns/dcat#"
+                xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+                exclude-result-prefixes="#all">
+
+    <xsl:output method="xml" indent="yes"/>
+    <xsl:strip-space elements="*"/>
+
+    <xsl:variable name="format-mimetype-mapping">
+        <entry>
+            <format>csv</format>
+            <mimetype>text/csv</mimetype>
+            <protocol>WWW:DOWNLOAD</protocol>
+        </entry>
+        <entry>
+            <format>geojson</format>
+            <mimetype>application/vnd.geo+json</mimetype>
+            <protocol>WWW:DOWNLOAD</protocol>
+        </entry>
+        <entry>
+            <format>kml</format>
+            <mimetype>application/vnd.google-earth.kml+xml</mimetype>
+            <protocol>WWW:DOWNLOAD</protocol>
+        </entry>
+        <entry>
+            <format>zip</format>
+            <mimetype>application/zip</mimetype>
+            <protocol>WWW:DOWNLOAD</protocol>
+        </entry>
+        <entry>
+            <format>shapefile</format>
+            <mimetype>x-gis/x-shapefile</mimetype>
+            <protocol>WWW:DOWNLOAD</protocol>
+        </entry>
+        <entry>
+            <format>json</format>
+            <mimetype>application/json</mimetype>
+            <protocol>WWW:DOWNLOAD</protocol>
+        </entry>
+        <entry>
+            <format>web page</format>
+            <mimetype>text/html</mimetype>
+            <protocol>WWW:LINK</protocol>
+        </entry>
+        <entry>
+            <format>arcgis geoservices rest api</format>
+            <mimetype>application/json</mimetype>
+            <protocol>ESRI:REST</protocol>
+        </entry>
```

Comment:
Actually now that we use this mapping in the ESRI transformation, this is redundant with:
https://github.com/geonetwork/core-geonetwork/blob/8a6f41ed61ccf0903f859e38d13d05471853ba6a/schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/convert/fromJsonLdEsri.xsl#L428-L429

I think the two lines above can be removed and the whole mapping logic can be done in `protocol-mapping.xsl`, that might be a good option?

Reply:
For example:
```xml
        <entry>
            <format>csv</format>
            <mimetype>text/csv</mimetype>
            <protocol>WWW:DOWNLOAD</protocol>
        </entry>
```

would become:

```xml
        <entry>
            <format>csv</format>
            <protocol>WWW:DOWNLOAD:text/csv</protocol>
        </entry>
```